source/user-manual/overview.rst

```diff
@@ -1,5 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: Wazuh is a comprehensive open source cybersecurity platform. In our user manual you can find out how to configure and use each of the Wazuh components. 
+  
```

Comment:
I can still see the old meta description under the title of this section (lines 8/10) and it needs to be removed.
If you've already fixed it later on, dismiss this comment.